runconfig/hostconfig_windows.go

```diff
@@ -31,7 +31,7 @@ func validateNetMode(c *container.Config, hc *container.HostConfig) error {
 	}
 
 	if hc.NetworkMode.IsContainer() && hc.Isolation.IsHyperV() {
-		return fmt.Errorf("net mode --net=container:<NameOrId> unsupported for hyperv isolation")
+		return fmt.Errorf("container network stack unsupported for hyperv isolation")
```

Comment:
I would write: Using the network stack of another  container is not supported while using Hyper-V Containers